internal/xds/translator/testdata/out/xds-ir/tracing-endpoint-stats.listeners.yaml

```diff
@@ -51,8 +51,7 @@
               '@type': type.googleapis.com/envoy.config.trace.v3.OpenTelemetryConfig
               grpcService:
                 envoyGrpc:
-                  authority: otel-collector.monitoring.svc.cluster.local
```

Comment:
why is authority removed ? is this intentional ?

Reply:
it's not, mistake by hand write input.